archaius2-core/src/main/java/com/netflix/archaius/DefaultAppConfig.java

```diff
@@ -205,46 +182,40 @@ public static Builder builder() {
         return new Builder();
     }
     
-    public static DefaultAppConfig createDefault() {
+    public static DefaultAppConfig createDefault() throws ConfigException {
         return new Builder().build();
     }
     
+    private final CascadingCompositeConfig     root;
     private final SimpleDynamicConfig          runtime;
```

Comment:
If I'm reading this correctly this is now the override layer. Can we name the variable override so it is easier to follow the code?